avm/ptn/azd/acr-container-app/main.bicep

```diff
@@ -0,0 +1,251 @@
+@allowed(['None', 'SystemAssigned', 'UserAssigned'])
+param identityType string = 'None'
+
+@description('Optional. The name of the container image.')
+param imageName string = ''
+
+@description('Optional. Specifies if Ingress is enabled for the container app.')
+param ingressEnabled bool = true
+
+@allowed([
+  'auto'
+  'http'
+  'http2'
+  'tcp'
+])
+@description('Optional. Ingress transport protocol.')
+param ingressTransport string = 'auto'
+
+@description('Optional. Rules to restrict incoming IP address.')
+param ipSecurityRestrictions array = []
+
+@description('Optional. Bool indicating if HTTP connections to is allowed. If set to false HTTP connections are automatically redirected to HTTPS connections.')
+param ingressAllowInsecure bool = true
+
+@allowed([
+  'Multiple'
+  'Single'
+])
+@description('Optional. Controls how active revisions are handled for the Container app.')
+param revisionMode string = 'Single'
+
+@description('Optional. The secrets required for the container.')
+@secure()
+param secrets object = {}
+
+@description('Optional. The service binds associated with the container.')
+param serviceBinds array = []
+
+@description('Optional. The name of the container apps add-on to use. e.g. redis.')
+param serviceType string = ''
+
+@description('Optional. The target port for the container.')
+param targetPort int = 80
+
+@description('Optional. Toggle to include the service configuration.')
+param includeAddOns bool = false
+
+@description('Optional. The principal ID of the principal to assign the role to.')
+param principalId string = ''
+
+@description('Optional. The resource id of the user-assigned identity.')
+param userAssignedIdentityResourceId string = ''
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+// Private registry support requires both an ACR name and a User Assigned managed identity
+var usePrivateRegistry = !empty(identityName) && !empty(containerRegistryName)
+
+// Automatically set to `UserAssigned` when an `identityName` has been set
+var normalizedIdentityType = !empty(identityName) ? 'UserAssigned' : identityType
+
+#disable-next-line no-deployments-resources
+resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableTelemetry) {
+  name: '46d3xbcp.ptn.azd-acrcontainerapp.${replace('-..--..-', '.', '-')}.${substring(uniqueString(deployment().name, location), 0, 4)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+      outputs: {
+        telemetry: {
+          type: 'String'
+          value: 'For more information, see https://aka.ms/avm/TelemetryInfo'
+        }
+      }
+    }
+  }
+}
+
+module containerApp 'br/public:avm/res/app/container-app:0.10.0' = {
+  name: '${uniqueString(deployment().name, location)}-container-app'
+  params: {
```

Comment:
Misses passthru of `enableTelemetry` parameter

Reply:
Hey @NanaXiong00, the fix was unfortunately not correct. It's about adding a new parameter to each module deployment's param block. **Not** a condition.
Also please don't self-resolve comments but just comment them once addressed. Makes re-reviews easier

Reply:
@AlexanderSehr I have updated by your comments.